src/pages/api/webhooks/webhook.ts

```diff
@@ -0,0 +1,39 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+interface IssuePayload {
+  issue: {
+    html_url: string;
+    title: string;
+  };
+  repository: {
+    name: string;
+  };
+}
+
+export default function webhooksHandler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    const eventType = req.headers['x-github-event'];
+    if (eventType !== 'issues') {
+      console.log(`Webhook ignored. Unsupported event type: ${eventType as string}`);
+      res.status(200).send('Webhook ignored');
```

Comment:
ignored is 200? feels like non-github-originated requests should return 401 or something, and requests for invalid event types should return something else

not a huge deal since this is just for testing tho

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses

Reply:
Yep. 200 wouldn't be the correct code but as you mentioned, it's not a concern as of right now since it's for testing purposes.